torchci/test/jobUtils.test.ts

```diff
@@ -565,4 +569,606 @@ describe("Test various job utils", () => {
+            html_url: "",
+          },
+        ],
+        "2024-05-06T00:00:00Z"
+      )
+    ).toEqual(false);
+
+    // The issue is close after the base commit date
+    expect(
+      isRecentlyCloseDisabledTest(
+        [
+          {
+            state: "closed",
+            number: 555,
+            title: "",
+            body: "",
+            updated_at: "2024-05-06T00:30:00Z",
+            author_association: "",
+            html_url: "",
+          },
+          {
+            state: "closed",
+            number: 666,
+            title: "",
+            body: "",
+            updated_at: "2024-05-06T01:00:00Z",
+            author_association: "",
+            html_url: "",
+          },
+        ],
+        "2024-05-06T00:00:00Z"
+      )
+    ).toEqual(true);
+  });
+
+  test("test getDisabledTestIssues", async () => {
+    // Invalid input should return nothing
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: [],
+        },
+        []
+      )
+    ).toEqual([]);
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: [],
+          name: "Anything goes",
+        },
+        []
+      )
+    ).toEqual([]);
+
+    // Having no disabled test issue
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: [
+            "test_cpp_extensions_open_device_registration.py::TestCppExtensionOpenRgistration::test_open_device_registration",
+          ],
+          name: "Anything goes",
+        },
+        []
+      )
+    ).toEqual([]);
+
+    // Not matching the failure regex
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: ["Not a failed test"],
+          name: "Anything goes",
+        },
+        [
+          {
+            state: "open",
+            number: 100152,
+            title:
+              "DISABLED test_open_device_registration (__main__.TestCppExtensionOpenRgistration)",
+            body: "Platforms: linux, win, mac",
+            updated_at: "2024-05-06T00:30:00Z",
+            author_association: "",
+            html_url: "",
+          },
+        ]
+      )
+    ).toEqual([]);
+
+    // Not matching test case
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: [
+            "test_cpp_extensions_open_device_registration.py::TestCppExtensionOpenRgistration::test_open_device_registration_no_match",
+          ],
+          name: "Anything goes",
```

Comment:
You might want to set the job names here to something that would match so that we can be sure the failure to match is due to the not matching test names